tests/wrappers/ReactWrapper-test.js

```diff
@@ -191,5 +193,15 @@ describe('ReactInVue', () => {
         )
       )
     })
+
+    it('works with no children', () => {
+      const mockError = jest.fn()
```

Comment:
Maybe we better use a [spy](https://facebook.github.io/jest/docs/en/jest-object.html#jestspyonobject-methodname)?

Reply:
Whoops, yes, thanks.